### ISSUE TITLE
Confusing output in Braille for Drum Kit Notation

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32804]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

Currently the output for unpitched percussion that is written with multiple instruments on a staff (such as for drum kit) is rather confusing in Braille.  It looks like the nearest white key note to the midi note is being used as the Braille letter to notate these on.  They are output on one line with in-accord sign between each voice, which makes good sense in a digital format that only shows 1 line and 1 bar at a time, rather than assigning each instrument to its own line which is the general way of transcribing them in a fully formatted Braille score.

In a Braille score the transcriber can choose the note letter they are assigned to, but they do have to include a list of these as a key at the top.  Most though generally adopt the system of matching them to what those lines and spaces would be read as on a bass clef.  So an instrument on bottom space would be Brailled as note A, top space G, ledger line above staff C, etc.  On MuseScore these would need to change in Braille to match if the user changes the space/line the instrument is assigned to. 

Although a blind user could memorise the midi letters associated with each instrument, the current output is hard to work with if a user is also referring to scores or putting the music through a Braille transcription that uses a more conventional mapping of the Braille letters for each instrument.

The Braille output is also missing the symbols before notes that refer to shape of note head and all other percussion specific markings.  At the least basic notehead shapes like diamond, cross, etc should be included.  In formatted braille scores these would show doubled before the first instance to indicate to be used until further notice.  In MuseScore a little creativity might be needed to be sensible and add them once before the first note they apply to for each pitch in a bar, a bit like an accidental works.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.7 beta

### Regression

No.

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
